ui/src/app/components/runs/runs.component.html

```diff
@@ -13,8 +13,64 @@
   ~ limitations under the License.
   -->
 
-<div class="content-container">
-  <div class="content-area">
-    <p>runs works!</p>
-  </div>
-</div>
+<clr-datagrid (clrDgRefresh)="onClarityDgRefresh($event)" [clrDgLoading]="loading">
+
+  <clr-dg-action-bar>
+    <div class="btn-group">
+      <button type="button" class="btn btn-sm btn-secondary"><clr-icon shape="trash"></clr-icon> Clear filters </button>
+      <button type="button" class="btn btn-sm btn-secondary"><clr-icon shape="trash"></clr-icon> Clear sort </button>
+      <button type="button" class="btn btn-sm btn-secondary"><clr-icon shape="refresh"></clr-icon> Refresh </button>
+    </div>
+  </clr-dg-action-bar>
+
+  <clr-dg-column [clrDgField]="'workflowName'" [clrFilterValue]>
```

Comment:
Here we have to think about how we can extract these hard-coded strings, since they have a dependency on the backend as well.

Reply:
Fixed. What do you think?

Reply:
Yeah, not bad. I think you should extract those field names in the backend as well from `DagRunTable`, maybe to `tables/constants/DagRunTableConstants` and then move `dagRunColumns.constants.ts` to `models/tables/constants/dagRunTableConstants.model.ts` or something like this. I think it's really important that we have a convention how we can find out which fields in the backend relate to which field in the frontend. Because already now with full-text search there are many results if you search for "workflowName" and I think as the project grows, it will slow us down when we need to add more fields to the table

Reply:
I agree with most of the things, but here I disagree. On backend side there is no need to extract all fields from tables into constants, they are used only on that one place and table class is the definition of the table. And you can clearly see relation between def column name and name in db ... On the frontend side, it is defined fine

Reply:
Ok let's leave it like this. My concern is, suppose we need to add another sort field in `DagRunTable`. How do you quickly know which frontend code you're gonna have to touch as well? If you know the project like us, it's probably no problem. But if someone new joins the project, he / she might have a hard time if we don't stick to strict conventions